assets/js/modules/analytics/common/timzezone-select.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * TimezoneSelect component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Select,
+	Option,
+} from '../../../material-components';
+import { countries } from './countries';
+import classnames from 'classnames';
+
+const { default: { country: allCountries } } = countries;
+
+const TimezoneSelect = ( { timezone, setTimezone, validationIssues, setValidationIssues } ) => {
+	const [ selectedCountry, setSelectedCountry ] = useState( timezone );
+	const [ selectedTimezoneID, setSelectedTimezoneID ] = useState( '' );
+	let multiTimezone,
+		selectedTimezoneDisplay = selectedTimezoneID;
+
+	const getTimezoneSelector = () => {
+		let foundTimezone = false;
+		const response = (
+			<div >
+				<span style={ { minWidth: '240px', margin: '0 5px 0 0' } /*todo: move to css */ } >
```

Comment:
Let's do this and eliminate the TODO :)

Reply:
👍

Reply:
Moved styles to Sass and improved in https://github.com/google/site-kit-wp/pull/1389/commits/0a9ee72421bc52d6f04ab325d5e384c265664f5c.